odoo/fields.py

```diff
@@ -1870,6 +1870,8 @@ def write(self, records, value):
                         if old_is_text or not closest_is_text:
                             if not closest_is_text and records.env.context.get("install_mode") and lang == 'en_US' and term_adapter:
                                 adapter = term_adapter(closest_term)
+                                if adapter(old_term) is None:
+                                    continue
                                 translation_dictionary[closest_term] = {k: adapter(v) for k, v in translation_dictionary.pop(old_term).items()}
```

Comment:
Don't we need this now?
```suggestion
                                translation_dictionary[closest_term] = {k: adapter(v) for k, v in translation_dictionary.pop(old_term).items() if adapter(v) is not None}
```

Reply:
Not really, for `field.translate` function translating `term_src` to `None`/` `/`<broken html`/`<div>muti</div>terms` is equal to translating `term_src` to `term_src`